mitodl/ocw-data-parser
# refactor master -> parsed and change output file name

#### What are the relevant tickets?
Fixes https://github.com/mitodl/ocw-data-parser/issues/76

#### What's this PR do?
This PR refactors the language around "master json" to be referenced as "parsed json."  It also changes the output file name to be the course id followed by "_parsed" like so:

`18-01-single-variable-calculus-fall-2005_parsed.json`

#### How should this be manually tested?
Parse a course and ensure that the outputted file matches the scheme above.

Note that this PR is blocked by a needed update to `open-discussions` and `ocw-to-hugo` to support the new output json filename.
